extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -54,6 +51,11 @@ class AmpSocialShare extends AMP.BaseElement {
     return true;
   }
 
+  /** @override */
+  prerenderAllowed() {
+    return true;
```

Comment:
I worry about the `urlReplacements.expandUrlAsync` below potentially doing XHR during prerender.

Let's split `buildCallback` in two steps: 1. building DOM/toggling and 2. setting `href`. 

The second step will `expandUrlAsync` once the document is visible. You'll need to check visibility state inside `buildCallback`. [See `amp-video.js` for reference.](https://github.com/ampproject/amphtml/blob/b80c1f2cf9be4189ca5eda479179251b3ccce61e/extensions/amp-video/0.1/amp-video.js#L325-L336)

Reply:
Discussed offline, moving this to `layoutCallback` instead